templates/requiredField.interface.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="required-field">
+  <button class="btn" data-remove-field><i class="fa fa-minus fa-fw fa-lg"></i></button>
```

Comment:
I can't compare them side by side using screencasts, but it seems that the minus icon is larger than what's used in the notifications UI. The button is also circular in the latter instead of a pill-shaped button. See https://github.com/Fliplet/fliplet-widget-push-notifications/blob/master/src/components/NotificationForm.vue#L66 for the markup.

Does it resolve both issues if you simply remove the `fa-lg`?

```suggestion
  <button class="btn" data-remove-field><i class="fa fa-minus fa-fw"></i></button>
```

Reply:
@tonytlwu doesn't look like 
![image](https://user-images.githubusercontent.com/574210/83157026-6b932280-a103-11ea-97f9-b955e20e8b65.png)

Reply:
Let's keep it without the `fa-lg` anyway because the + and - icons are much more similar in size this way. If you can, take a look to see why it isn't perfectly circular like the notifications UI, but I'm not that bothered as long as it doesn't use `fa-lg`.